packages/server/postgres/queries/getUserIdsToPause.ts

```diff
@@ -0,0 +1,13 @@
+import {getUserIdsToPauseQuery} from './generated/getUserIdsToPauseQuery'
+import getPg from '../getPg'
+import catchAndLog from '../utils/catchAndLog'
+
+const getUserIdsToPause = async (activeThreshold: Date): Promise<string[]> => {
+  return (
```

Comment:
`returns await` is actually unnecessary. I recently learnt this from [Matt's comment](https://github.com/ParabolInc/parabol/pull/5201#discussion_r695311678). There's also an [ESLint rule](https://eslint.org/docs/rules/no-return-await).

Reply:
Sure, saw this used in other places and it's useful for stack traces, but that's also mentioned in the lint rule, so we know the trade off.

Reply:
Wanted to change it, it's actually `(await ...) ?? []`, so I cannot remove it.